Source/csla.netcore.test/BasicModern/Child.cs

```diff
@@ -27,7 +27,7 @@ public string Name
       set { SetProperty(NameProperty, value); }
     }
 
-    public void Child_Create(int id, string name)
+    private void Child_Create(int id, string name)
```

Comment:
For all the operations on this class, should they have the new operation attributes on them?

Reply:
Yeah, probably should. Will do, at least in all the newer dotnet core tests.